### DISCUSSION TITLE
prestação de serviços

### DISCUSSION THREAD

**Discussion by i7system** [wmixvideo/nfe#725] [Geral] [answered]
bom dia , meu nome é paulo sanches  e gostaria de saber alguem quer ganhar um dinheiro extra  , rs

preciso de umas dicas de como usar seu sistema de nfe.

me chama no zap : 13-98124-3260

pode chamar a qq horas,

obrigado.

**Comment by fincatto** [comment, marked as answer]
Boa tarde @i7system. 
Espero que tenhas conseguido o suporte de que precisava.
Muitos de nós trabalhamos para empresas que usam todo (ou quase) todo o nosso tempo para as suas demandas, então nem sempre é possível pegar projetos paralelos.
